src/bundle/Resources/public/js/CKEditor/custom-attributes/ui/custom-attributes-form-view.js

```diff
@@ -209,8 +209,25 @@ class IbexaCustomAttributesFormView extends View {
         addListToDropdown(labeledDropdown.fieldView, itemsList);
 
         this.listenTo(labeledDropdown.fieldView, 'execute', (event) => {
-            const alreadySelectedValues = labeledDropdown.fieldView.element.value ? labeledDropdown.fieldView.element.value.split(' ') : [];
-            const value = config.multiple ? [...new Set([...alreadySelectedValues, event.source.value])].join(' ') : event.source.value;
+            const previousValue = labeledDropdown.fieldView.element.value;
+            let value;
+
+            if (config.multiple) {
+                value = previousValue ? [...new Set([...previousValue.split(','), event.source.value])].join(',') : event.source.value;
```

Comment:
you probably reintroduced bug which I fixed lately https://issues.ibexa.co/browse/IBX-5981

Reply:
@dew326 I actually cannot spot any problem when `default_value` is not present in configuration

Reply:
@dew326 the following config seems to work propery:
```
                    classes:
                        paragraph:
                            choices: [ test1, test2, test3 ]
                            #default_value: test1
                            required: false
                            multiple: true
```
am I missing something?

Reply:
https://github.com/ibexa/fieldtype-richtext/pull/95#issuecomment-1603733713

Reply:
@dew326 ah, so the problem was about commas, hopefully this is what you had in mind: https://github.com/ibexa/fieldtype-richtext/pull/104/commits/ce4b41b4d693c40d09b4118b66a2adb2329e66c3